edx_solutions_api_integration/utils.py

```diff
@@ -485,6 +485,15 @@ def get_image_dimensions(image_url):
         return img.size
 
 
+def exclude_non_actual_company_users(users_queryset, exclude_type, actual_organizations=[]):
+    """
+    excludes users which are not part of an actual organization
+    """
+    admin_users = list(users_queryset.filter(groups__groupprofile__name=exclude_type))
```

Comment:
This query can slow down the API. Can we cache company admin user ids the same way we cache users with roles in `get_aggregate_exclusion_user_ids`?

Reply:
Addressed